packages/devtools_app/lib/src/logging/logging_screen.dart

```diff
@@ -413,6 +415,20 @@ class _MessageColumn extends ColumnData<LogData>
   String getValue(LogData dataObject) =>
       dataObject.summary ?? dataObject.details;
 
+  @override
+  int compare(LogData a, LogData b) {
+    final String valueA = getValue(a);
+    final String valueB = getValue(b);
+    // Matches frame descriptions (e.g. '#12  11.4ms ')
+    final regex = RegExp(r'#\d+\s+\d+.\d+ms\s*');
+    if (valueA.startsWith(regex) && valueB.startsWith(regex)) {
+      final frameNumberA = valueA.substring(1, valueA.indexOf(' '));
```

Comment:
safer to have a match in the regexp using `(` and `)` so the part you match in the reg-exp is exactly the # you expected.

Reply:
done